kinetic/distribution.yaml

```diff
@@ -5085,6 +5085,23 @@ repositories:
       url: https://github.com/christianrauch/msp.git
       version: master
     status: developed
+  multi_jackal-release:
```

Comment:
repository names usually match the name of the source repository. So in this case it would be `multi_jackal` rather than `multi_jackal-release`